src/components/evaluation/Chart.tsx

```diff
@@ -9,29 +9,17 @@ import PieChart from './charts/PieChart'
 import TableChart from './charts/TableChart'
```

Comment:
The generic chart components (and all child charts) do not need to know anything about the feedback or confusion stuff, as it is a higher level concept (and handled in evaluation.tsx).